public/javascripts/data.js

```diff
@@ -706,8 +706,7 @@ var dataNS = odkmaker.namespace.load('odkmaker.data');
             name: 'data',
             attrs: {
               'id': $.sanitizeString($('.header h1').text()),
-              'version': 'build_' + $.sanitizeString($('.header h1').text()) +
-                    '_' + Math.round((new Date()).getTime() / 1000)
+              'version': Math.round((new Date()).getTime() / 1000)
```

Comment:
Addressing https://github.com/getodk/build/issues/236#issuecomment-736995794

Reply:
Version needs to be `'' + Math.round to become a string else XML export fails when it gets the date hash as number.`

Reply:
i might mildly prefer `String(Math.round(…))` over `'' +` but tbh the latter is probably the more common pattern anyway.